session replay product docs (#6255)
getsentry/sentry-docs: src/docs/product/session-replay/replay-details.mdx

---
title: "Replay Details"
sidebar_order: 24
description: "Session Replay Details."
---

<Include name="beta-note-session-replay.mdx" />

Every replay has a detailed view that contains the embedded video player and rich debugging context. Playing back the video will allow you to see every user interaction in relation to console messages, DOM events, and network requests. It’s like having [DevTools](https://developer.chrome.com/docs/devtools/overview/) active for your production user sessions. Almost every component on this page is connected through timestamps. See the breakdown of each component and why it’s valuable:

> - **Replay Player:** Video-like reproduction of a user session created from DOM snapshots. This is where you can visualize exactly what actions the user took during a user session, and how the website behaved within this specific user’s environment, including browser, latency, settings, and so on. Most importantly, you can see which actions lead to an error and take the guesswork out of debugging.Note that by default, the Session Replay SDK is configured to replace all text nodes and user input with asterisks (*), and to remove media elements. See all privacy configuration options [here](/platforms/javascript/session-replay/privacy/).

Reviewer: ```suggestion
- **Replay Player:** Video-like reproduction of a user session created from DOM snapshots. This is where you can visualize exactly what actions the user took during a user session, and how the website behaved within this specific user’s environment, including browser, latency, settings, and so on. Most importantly, you can see which actions lead to an error and take the guesswork out of debugging.Note that by default, the Session Replay SDK is configured to replace all text nodes and user input with asterisks (*), and to remove media elements. See all privacy configuration options [here](/platforms/javascript/session-replay/privacy/).

```

Author: Added as sub items. Removed the "Note that" prefix from all.

- **Network:** A list of all network requests that were initiated by the page while the replay recording was active. As the video plays, there’s a visual indicator that tracks through the table of network requests, highlighting which requests happened prior to, or next to this point in the video. You can also click the timestamp on the far right of each request to bring yourself to that point in the replay player.
- **DOM Events:** DOM event breadcrumbs, like user clicks, are augmented with surrounding HTML source code to help users understand which event was targeted. As the video plays, there’s a visual indicator that tracks through the table of DOM events, highlighting which events happened prior to, or next to this point in the video. You can also click the timestamp on the far right of each DOM event to bring yourself to that point in the replay player.

> - **Trace:** Connects all the [trace(s](/product/sentry-basics/tracing/distributed-tracing#traces-transactions-and-spans)) that happened during the replay. Note: Due to transaction sampling, this view may be missing traces.

Reviewer: @bruno-garcia

> - **Trace:** Connects all the [trace(s](/product/sentry-basics/tracing/distributed-tracing#traces-transactions-and-spans)) that happened during the replay. Note: Due to transaction sampling, this view may be missing traces.

Author: ```suggestion
- **Trace:** Connects all the [trace(s)](/product/sentry-basics/tracing/distributed-tracing#traces-transactions-and-spans) that happened during the replay. Note: Due to transaction sampling, this view may be missing traces.
```

- **Issues:** See the [issue(s)](/product/issues/issue-details/error-issues/) that correspond to frontend errors that occurred in the replay and the impact these issues have had holistically across *all users* on your application.
- **Memory:** The view shows a heap size chart displaying the total amount of memory being used by JavaScript objects.

<Note>

> This view is only available for Chromium-based browsers.

Author: How about we go with this instead:

```suggestion
This view is only available when the user session was recorded on a Chromium-based browser.
```
